CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1051,7 +1051,7 @@ void __CFInitialize(void) {
         #if TARGET_OS_LINUX
         #define __CFSwiftGetBaseClass _TF10Foundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
         #elif TARGET_OS_MAC
-        #define __CFSwiftGetBaseClass _TF15SwiftFoundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
+        #define __CFSwiftGetBaseClass _TF18PortableFoundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
```

Comment:
Yes, just changed the name. You're absolutely right that unifying the names is important - what about if you need to use the module name to disambiguate something for the swift compiler?

e.g. I can declare my own type called NSString, and to access Foundation's type I refer to `Foundation.NSString`. If the module names aren't the same, I'm going to have to duplicate that OS-switching logic that normally only happens at import. e.g:
```
class NSString {} // This ain't Obj-C; we've got namespaces now!
#if os(OSX)
   let foundationString = SwiftFoundation.NSString()
#else
   let foundationString = Foundation.NSString()
#endif
```

This isn't a problem now, because we're still using Objective-C-style prefixed names. If we were to drop the prefixes (https://github.com/apple/swift-evolution/blob/master/proposals/0086-drop-foundation-ns.md), these name conflicts would likely occur more often and we'd have to do this ugly switching whenever it happens. That's made harder by the fact that `#if os` isn't a macro - it needs to take a whole thing, like an entire function or class.